pkg/apis/core/v1alpha1/types.go

```diff
@@ -28,6 +28,7 @@ import (
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
 // +kubebuilder:object:root=true
 // +kubebuilder:resource:shortName=dv;dvs,categories=all
+// +kubebuilder:subresource:status
```

Comment:
What does this do?  If I recall, we do not have subresource status enabled on our CRDs.

Reply:
Discussed it with @awels.
Part of the patch was enabling status subresource, which is the right way to update status only.

Reply:
okay I see you added the status subresource